server/methods/authentication/plexAuthentication.js

```diff
@@ -32,21 +32,26 @@ Meteor.methods({
             }
         }
 
+        //If it is in the array, will return the username, if not, returns false
         function isInArray(value, array) {
-		  return array.indexOf(value) > -1
+          for(var i = 0; i < array.length; i++) {
```

Comment:
Could alternatively use map here to keep from nesting loops

```javascript
var usernames = array.map(function(user) {
  return user.split(':');
}
```

That is unless we are required to support IE8.

Reply:
Good note, I didn't notice the use of map there!